index.js

```diff
@@ -187,12 +183,34 @@ ShareDbMongo.prototype._connect = function(mongo, options) {
     mongo(finish);
     return;
   }
-  // TODO: Don't pass options directly to mongodb.connect();
-  // only pass options.mongoOptions
```

Comment:
IIRC, we were planning on deprecating / removing the ability to pass in the Mongo driver options at the top level options, instead requiring that they be passed via the `mongoOptions` option.

E.g.:
```
// Deprecated
new ShareDbMongo(url, { readPreference: 'secondaryPreferred' });
// Recommended
new ShareDbMongo(url, { mongoOptions: { readPreference: 'secondaryPreferred' } });
```

If we still want to do that, to avoid all the options clean-up below, then we should leave a comment as a reminder for us to do it.

Reply:
Good point; I've put the TODO back